src/comps/Specials/Netherland.svelte

```diff
@@ -31,43 +29,45 @@
 
             return {
                 id: product.id,
-                nr: ((i + 1) + '').padStart(2, '00'),
+                nr: (i + 1 + '').padStart(2, '00'),
                 title: product.title.replace(STR_NETHERLAND + ' ', ''),
                 state: getEEState(product),
-            }
-        })
+            };
+        });
     }
 
     const setActive = (event, id) => {
         event.stopPropagation();
         activeProductID = id;
 
-        storedActiveProduct.update(store => {
+        storedActiveProduct.update((store) => {
             store.product = {
                 id: activeProductID,
                 type: 'products',
             };
             store.reason = 'click-on-zoom';
             return store;
-        })
-    }
+        });
+    };
 </script>
 
-<svelte:window bind:innerWidth={innerWidth}/>
```

Comment:
na ist das going?

Reply:
probably. what shall it do anyway?

Reply:
https://svelte.dev/tutorial/svelte-window attribute name vs variable binding....